<launch>
    <node name="audit_image" pkg="ros_audit_image" type="audit_image.py" output="screen">
        <rosparam command="delete" param=""/>
        <rosparam command="load" file="$(find ros_audit_image)/config/config.yaml"/>
    </node>
</launch>
